### PR TITLE
Fix scene creation

### PR DESCRIPTION
**Pull request by guycalledfrank**
Also change preRender event name

### CODE REVIEW

File: src/framework/application.js [hunk 1]
```diff
@@ -381,14 +371,6 @@ pc.extend(pc, function () {
                             self.systems.rigidbody.setGravity(scene._gravity.x, scene._gravity.y, scene._gravity.z);
                         }
 
-                        if (!first) {
```
Comment (daredevildave) [inline src/framework/application.js]: err, don't think that is right...

Comment (daredevildave) [inline src/framework/application.js]: we need this bit?

Comment (daredevildave) [inline src/framework/application.js]: sorry, my mistake...

Comment (guycalledfrank) [inline src/framework/application.js]: It was the accounting for the case when scene is created multiple times.

**Pull request merged**